articles/azure-large-instances/work-with-ali-in-the-azure-portal.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Working with ALI in the Azure portal
+description: Shows how to what you can do in the Azure portal with ALI.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.custom: devx-track-azurecli
+ms.date: 06/01/2023
+---
+
+# Working with  ALI in the Azure portal 
+
+In this article, we'll show what you can do in the Azure portal with your implementation of Azure Large Instances (ALI).
+
+> [!Note]
+> For the time being, you will notice the BareMetal Infrastructure naming convention still in use in the Azure portal. This will be replaced with ALI soon. Until then, BareMetal Infrastructre or BareMetal Instances can be considered synonyms for ALI.
+
+## Register the resource provider
+
+An Azure resource provider for ALI enables to you see the instances in the Azure portal.
```

Comment:
Sentence needs to be fixed as "An Azure resource provider for ALI enables **you to see** the instances in the Azure portal

Reply:
Fixed 
To confirm in build